src/main/scala/net/ripe/rpki/publicationserver/parsing/StaxParser.scala

```diff
@@ -50,8 +50,7 @@ case class UnknownEvent(code: Int) extends StaxEvent
 object StaxEvent {
```

Comment:
Do we need to change some instance creations or something?

Reply:
No, the actual reader is passed to the function and not shared across threads.

Reply:
awesome